clusterloader2/pkg/chaos/nodes.go

```diff
@@ -124,7 +124,16 @@ func (k *NodeKiller) kill(nodes []v1.Node) {
 			time.Sleep(time.Duration(k.config.SimulatedDowntime))
 
 			klog.Infof("%s: Rebooting %q to repair the node", k, node.Name)
-			err = util.SSH("sudo reboot", &node, nil)
+			// Scheduling a reboot in one second, then disconnecting.
+			//
+			// Bash command explanation:
+			// 'nohup' - Making sure that end of SSH connection signal will not break sudo
+			// 'sudo' - Elevated priviliages, required by 'reboot'
+			// 'reboot' - Rebooting node
+			// '+1s' - Parameter to 'reboot', to wait 1 second before rebooting.
+			// '> /dev/null 2> /dev/null < /dev/null' - File descriptor redirect, as we are closing terminal
```

Comment:
The explanation is not very clear. Wouldn't be better to write something like:
```
// redirect all three I/O to avoid ssh hanging, see https://web.archive.org/web/20090429074212/http://www.openssh.com/faq.html#3.10
```
?

Reply:
Added this comment